compiler-core/src/type_.rs

```diff
@@ -533,6 +533,15 @@ impl ModuleInterface {
             .cloned()
             .collect_vec()
     }
+
+    pub fn used_name(&self) -> SmolStr {
+        SmolStr::from(
+            self.name
+                .split('/')
+                .last()
+                .expect("Could not identify imported module name."),
```

Comment:
This code now panics even when the data is valid. Please have it return an optional.

Reply:
Sorry, I don't understand this. In what situation would it panic? The tests are passing, so I don't see how this could be true.